### PR TITLE
Update iOS sdk instructions for Carthage support

### PR DESCRIPTION
**Pull request by mspensieri**
@alavers @jpjoyal @Mario54 @jugarrit

### CODE REVIEW

File: source/includes/_adding_smooch_ios.md [hunk 1]
```diff
@@ -1,39 +1,40 @@
 # Adding Smooch to your app
 
-There are two ways to install the Smooch SDK. The recommended and easier way makes use of [CocoaPods](http://cocoapods.org) to manage library dependencies and SDK updates. Alternatively, you can go the manual route and drop the SDK into your Xcode project.
+There are two ways to install the Smooch SDK. The recommended and easier way is through [CocoaPods](#cocoapods-method). Alternatively, you can go the [manual route](#manual-method--carthage) and drop the SDK into your Xcode project.
```
Comment (jpjoyal) [inline source/includes/_adding_smooch_ios.md]: why lump manual & Carthage together ?

Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: Integration instructions are the exact same, the only thing Carthage does for you is download the framework

**Comment by gozman** [comment]
Should we update web app iOS sdk instructions?

File: source/includes/_adding_smooch_ios.md (hunk 1 above)
Comment (jpjoyal) [inline source/includes/_adding_smooch_ios.md]: I would phrase it like `Our recommended way to install Smooch is through [CocoaPods](#cocoapods-method). We support Carthage too and you can also go the [manual route](#manual-method--carthage) and drop the SDK into your Xcode project.`

Comment (mspensieri) [inline source/includes/_adding_smooch_ios.md]: Done

**Comment by jpjoyal** [comment]
:+1:

**Pull request merged**